tests/helpers/bootstrap.js

```diff
@@ -28,6 +29,23 @@ export function versionDependent(v3, v4) {
   return v4;
 }
 
+// eslint-disable-next-line ember/no-test-module-for
+export function moduleForOptionalFeature(optionalFeature, fn) {
+  let ownConfig = getOwnConfig();
```

Comment:
Something goes wrong here in an Embroider build.

This is the build output:

```js
  let ownConfig = Object(_home_jelhan_Code_ember_ecosystem_ember_bootstrap_node_modules_embroider_macros_src_addon_runtime__WEBPACK_IMPORTED_MODULE_2__["config"])("/tmp/embroider/6e5bc6/tests/dummy");
```

It's working fine if changing it manually to this:

```js
  let ownConfig = Object(_home_main_Code_ember_ecosystem_ember_bootstrap_node_modules_embroider_macros_src_addon_runtime__WEBPACK_IMPORTED_MODULE_2__["config"])("/tmp/embroider/6e5bc6")
```

That variable is called `packageRoot`. So it seems as if `@embroider/macros` uses a wrong package root for dummy app in an embroider build.

Will report upstream.

Reply:
@jelhan you already have the changes applied you mentioned in the embroider issue (`getConfig('ember-bootstrap')`), right?  Could you please push this, so we have this most current version available here (that fails for a classic build instead of embroider if I correctly understood), as a reproduction in case I find the time to look at this!

Reply:
I pushed a working solution for both classic and embroider builds in f23191f3cfba12d43196a12090eee75be207c025. It uses the correct code for embroider builds and uses the wrong behavior of `getOwnConfig` in classic builds. Would you be okay with using that work-a-round for now to unblock that one? We can remove it by deleting a few lines of code as soon as the bug is fixed upstream. It also only affects our own tests. So no risk for consumers.

Reply:
:+1: